File: src/main/java/com/example/ex3_2_back/entity/GenreHub.java
package com.example.ex3_2_back.entity;


import jakarta.persistence.*;
import lombok.*;

@Entity
@Table(name = "genre_hub")
@Builder
@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
public class GenreHub {
    @Id
    @Column(name = "genre_hub_id")
    @GeneratedValue(strategy = GenerationType.AUTO)
    Integer id;
    String name;
}

File: src/main/kotlin/com/example/ex3_2_back/entity/GenreHub.kt
package com.example.ex3_2_back.entity

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table(name = "genre_hub")
class GenreHub(
    @Id
    @Column(name = "genre_hub_id")
    var id: Int = 0,
    var name: String = "",
)